README.md

```diff
@@ -2,7 +2,7 @@
 <img src="https://github.com/dmmulroy/minttea/assets/2755722/e9e96e73-1f7f-4b8f-8bb1-445308dfe8bd" alt="Mint Tea Logo" width="400"/>
 
 A fun, functional, and stateful way to build terminal apps in OCaml heavily
-inspired by [BubbleTea][bubbletea]. MintTea is built on Riot and uses The Elm
+inspired by [BubbleTea][bubbletea]. Mint Tea is built on Riot and uses The Elm
```

Comment:
Considered updating this to "Bubble Tea" as well (inserting a space). It seems like that aligns more directly with their documentation. Thoughts?